pkg/cloud/option.go

```diff
@@ -0,0 +1,27 @@
+package cloud
+
+// Option are optional parameters to the generated methods.
+type Option struct {
```

Comment:
naive question, where are the parameters defined?

Reply:
Made the implementation much more clear.